WIP - improve synopsises (#931)
cosmos/sdk-tutorials: academy/2-main-concepts/context.md

* [Queries](./queries.md)
* [Events](./events.md)

> Context is the setting in which transactions execute. It is the sum of all pertinent information at runtime. Find out what this means in detail and the important elements that together form the execution context.

Reviewer: ```suggestion
Context is the setting in which transactions execute and is the sum of all pertinent information at runtime. Find out what transaction context means in detail and learn more about the important elements that together form the execution context.
```
